src/components/PublishArticle.jsx

```diff
@@ -23,6 +23,8 @@ const PublishArticle = (props) => {
           category: document
             .getElementById("category")
             .firstElementChild.innerText.toLowerCase(),
+          location: e.target.local.checked && "Sweden",
```

Comment:
Looks clever. Have you checked with a debugger what param is sent off?

Reply:
I checked now, in the local API log, we receive the param `location: "Sweden` 😎  Nice job!